R/mrmash_wrapper.R

```diff
@@ -356,4 +404,41 @@ autoselect_mixsd <- function(gmin, gmax, mult=2){
     npoint = ceiling(log2(gmax/gmin)/log2(mult))
     return(mult^((-npoint):0) * gmax)
   }
+}
+
+
+
+#' @importFrom mr.mash.alpha compute_univariate_sumstats
+# Compute Bhat and Sbhat summary statistics from univariate regression 
+# cv_sumstat <- function(X, Y, partition, standardize, nthreads){
+    
+#     ###Get fold names
+#     folds <- sort(unique(partition$fold))
+    
+#     ###List to store results
+#     res_all_folds <- list() 
+
+#     for(i in folds){
+#         test_ids <- partition[which(partition$fold == i), 1]
+#         Xtrain <- X[!(rownames(X) %in% test_ids), ]
+#         Ytrain <- Y[!(rownames(Y) %in% test_ids), ]
+
+#         univ_sumstats <- compute_univariate_sumstats(X=Xtrain, Y=Ytrain, standardize=standardize,
+#                                                                 standardize.response=FALSE,
+#                                                                 mc.cores=nthreads)
+#         res_all_folds[[paste0("fold_", i)]] <- univ_sumstats
+        
+#     }
+    
+#     return(res_all_folds)
+    
+# }
+
+
+#' @importFrom mr.mash.alpha compute_univariate_sumstats
+sumstat <- function(X, Y, standardize, nthreads){
```

Comment:
this wrapper function seems not useful and the function name is also confusing (it is not a verb). Please remove this wrapper and simply use `compute_univariate_sumstats` unless you have a good reason to not do that.

Reply:
@Chunmingl here is a comment you havent yet addressed

Reply:
it's now updated